src/localization/relocalizers/lsh_cv_hashing.cpp

```diff
@@ -97,17 +97,17 @@ std::vector<int> LshCvHashing::getCandidates(int quId) {
     exit(EXIT_FAILURE);
   }
   printf("Getting candidates for a query image\n");
-  const auto featurePtr = std::static_pointer_cast<const iFeature>(
+  const auto &feature = static_cast<const iFeature&>(
       _database->getQueryFeature(quId));
-  if (!featurePtr) {
-    printf("Wrong feature format\n");
-  }
+  // if (!featurePtr) {
```

Comment:
nit: it's usually better to remove the comments and rely on the versioning system to maintain the history of the code changes. Did you mean to leave this comment here?

Reply:
Neh, not really. Just forgot. Thanks.